lib/engine.py

```diff
@@ -701,6 +678,37 @@ def warning(self, warning):
         new_warning = Warning('%s:%s' % (self.script.shortname, self.table.name), warning)
         self.warnings.append(new_warning)
 
+    def load_data(self, filename):
+        """Generator uses the csv module to open a file and returns a list.
+
+        Opens a file and gets and sets the delimiter.
+        opens a file using the csv module and removes any extra line endings in the
+        fields
+        opens fixed width files and returns the list of values using the fixed width as the
+        delimiter
+        """
+        reg = re.compile("\\r\\n|\n|\r")
+        if not self.table.delimiter:
+            with io.open(filename, newline='', encoding='latin-1') as dataset_file:
```

Comment:
Newline controls how universal newline characters.
In this case ""  for Universal mode is used when empty "".
Otherwise we can set it to something different